apps/ledger-live-mobile/e2e/models/market/marketPage.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  getElementById,
+  getElementByText,
+  typeTextByElement,
+  tapByText,
+  tapByElement,
+} from "../../helpers";
+
+export default class marketPage {
+  searchBar = () => getElementById("search-box");
+  starBtn = () => getElementById("starAsset");
```

Comment:
Looks good. Just to keep things formatted consistently:
- name functions like `starButton` rather than `starBtn`. Just makes it a bit clearer
- use [kebab-case ](https://www.theserverside.com/definition/Kebab-case) like `star-asset` instead for testIDs